code/game/objects/items/props/helmetgarb.dm

```diff
@@ -496,6 +496,32 @@
 	desc = "The USCM had its funding pulled for these when it became apparent that not every deployed enlisted was wearing a helmet 24/7; much to the bafflement of UA High Command."
 	icon_state = "helmet_gasmask"
 
+/obj/item/prop/helmetgarb/helmet_gasmask/on_enter_storage(obj/item/storage/internal/helmet_internal_inventory)
+	if(!istype(helmet_internal_inventory))
+		return
+
+	var/obj/item/clothing/head/helmet/helmet_item = helmet_internal_inventory.master_object
+
+	if(!istype(helmet_item))
+		return
+
+	helmet_item.flags_inventory|=BLOCKGASEFFECT
+	helmet_item.flags_inv_hide|=HIDEFACE
+	..()
+
+/obj/item/prop/helmetgarb/helmet_gasmask/on_exit_storage(obj/item/storage/internal/helmet_internal_inventory)
+	if(!istype(helmet_internal_inventory))
+		return
```

Comment:
This test is likely unnecessary since the parent call isn't performing any test like this.

You should be able to remove these lines without issue. If this actually can occur, we would probably rather it runtime so the cause can be fixed rather than just testing and ignoring silently.